# Generated by Django 5.1.6 on 2025-04-25 13:23

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('secmomo', '0008_alter_agents_email'),
    ]

    operations = [
        migrations.AddField(
            model_name='agentapplication',
            name='username',
            field=models.CharField(blank=True, max_length=50, null=True, unique=True),
        ),
        migrations.AlterField(
            model_name='agents',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
        migrations.AddConstraint(
            model_name='agentapplication',
            constraint=models.UniqueConstraint(fields=('username',), name='unique_application_username'),
        ),
    ]
